website/components/ApiMeta.tsx

```diff
@@ -49,6 +50,11 @@ export function ApiMeta(props: ApiMetaProps) {
           Stability: {props.stability}
         </div>
       )}
+      {props.specific && (
+        <span className={`${tagStyle} ${styles.specific}`}>
+          <a href={href}>{props.specific.join('/')} specific</a>
```

Comment:
The `specific` tag should not link to `/misc/planning/future`, can we link to another page, or just remove the link